src/override/malloc-extensions.cc

```diff
@@ -6,10 +6,8 @@ using namespace snmalloc;
 
 void get_malloc_info_v1(malloc_info_v1* stats)
 {
-  auto unused_chunks =
-    Globals::get_handle().get_slab_allocator_state().unused_memory();
-  auto peak =
-    Globals::get_handle().get_slab_allocator_state().peak_memory_usage();
+  auto unused_chunks = Globals::get_slab_allocator_state().unused_memory();
```

Comment:
Do we need to guard against the case where this requires a parameter to `get_slab_allocator_state`?  I guess in those cases don't have a default?